src/NzbDrone.Common/Instrumentation/CleanseLogMessage.cs

```diff
@@ -20,7 +20,7 @@ public class CleanseLogMessage
                 new Regex(@"getnzb.*?(?<=\?|&)(r)=(?<secret>[^&=]+?)(?= |&|$)", RegexOptions.Compiled | RegexOptions.IgnoreCase),
                 
                 // Trackers Announce Keys; Designed for Qbit Json; should work for all in theory
-                new Regex(@"announce(\.php)?(/|%2f|%3fpasskey%3d)(?<secret>[a-z0-9]{16,})|(?<secret>[a-z0-9]{16,})(/|%2f)announce"),
+                new Regex(@"announce(\.php)?(/|%2f|%3fpasskey%3d|\?passkey=)(?<secret>[a-z0-9]{16,})|(?<secret>[a-z0-9]{16,})(/|%2f)announce"),
```

Comment:
Doesn't do anything, coz `passkey=xxx` was already handled. Did you even test it?

Reply:
Re-tested locally w/o the change;  correct is already handled

however, per https://github.com/Sonarr/Sonarr/issues/4623#issuecomment-897265596

it was not actually cleansed.  Why does the testcase pass yet the logs are not actually cleansed?

Reply:
resolved on discord; reverted this change; updated first regex `(?<secret>[^&="]+?)(?=[ "&=]|$)` as discussed

testcase improved